topgun/k8s/k8s_suite_test.go

```diff
@@ -30,8 +29,8 @@ func TestK8s(t *testing.T) {
 }
 
 type environment struct {
-	ChartsDir            string `env:"CHARTS_DIR,required"`
-	ConcourseChartDir    string `env:"CONCOURSE_CHART_DIR"`
+	HelmChartsDir        string `env:"HELM_CHARTS_DIR,required"`
+	ConcourseChartDir    string `env:"CONCOURSE_CHART_DIR,required"`
```

Comment:
won't the `HELM_CHARTS_DIR` and the `CONCOURSE_CHART_DIR` be the same thing, or is this for the sake of other charts? 🤔

Reply:
In some tests we use other charts (postgres and prometheus) from the helm charts repo. 
1. When we test metrics emitting
2. when we test using an external postgres database.

`ConcourseChartDir` always points to the root of the concourse helm chart and `ChartsDir` always points to the root of the helm repo.